vllm/model_executor/model_loader/loader.py

```
@@ -110,6 +110,11 @@ def _initialize_model(
     model_config = vllm_config.model_config
     model_class, _ = get_model_architecture(model_config)
 
+    # pass packed_modules_mapping by reference to quant_config
+    packed_mapping = getattr(model_class, "packed_modules_mapping", None)
```

Some models don't have `packed_modules_mapping ` yet, should we throw an exception?

---

In the case that the mapping is not defined, then the model is only liable to fail with quantization or lora. I see no reason to throw an error, as this will affect users who are not using quantization or lora

---

Oh,  error  indeed should not be raised here. When using quantization, appropriate exception handling should be added.

---

@jeejeelee Is checking for `vllm_config.quant_config is not None` a reliable way of checking if quantization is being used and therefore if an exception should be raised?

---

Perhaps a warning and not exception, as this is mostly a problem for users who have modules in the ignore list that are affected by fusing